src/electron/tunnel_store.ts

```diff
@@ -92,6 +92,6 @@ export class TunnelStore {
     if (!config || !tunnel.id) {
       return false;
     }
```

Comment:
This should be a `ShadowsocksSessionConfig`. Also, I believe we don't need the name (to check).